src/i18n/content-types.ts

```diff
@@ -53,6 +53,28 @@ interface Navbar {
 			changePassword: string;
 			deleteAccount: string;
 			approve: string;
+			usernameShould: string;
+			checkInput: string;
+			validateLength: string;
+			onlyNumberAndLetters: string;
+			userOccupied: string;
```

Comment:
let's try to stick to one way of naming things, in the component this is named `usernameTaken`, i'd suggest to use the same name here